### DISCUSSION TITLE
el-tour的按钮问题

### DISCUSSION THREAD

**Discussion by Xmppp** [element-plus/element-plus#19302] [Q&A] [answered]
我想改变【上一步】【下一步】按钮的样式和文字，
![微信图片编辑_20241217154304](https://github.com/user-attachments/assets/4e4656a8-4c90-4970-8032-ee9a1b988469)
目前只找到怎么修改样式，但不能修改文字。有什么办法么？

**Comment by rzzf** [comment, marked as answer]
[例子](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCIgQGNsaWNrPVwib3BlbiA9IHRydWVcIj5CZWdpbiBUb3VyPC9lbC1idXR0b24+XG5cbiAgPGVsLWRpdmlkZXIgLz5cblxuICA8ZWwtc3BhY2U+XG4gICAgPGVsLWJ1dHRvbiByZWY9XCJyZWYxXCI+VXBsb2FkPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbiByZWY9XCJyZWYyXCIgdHlwZT1cInByaW1hcnlcIj5TYXZlPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbiByZWY9XCJyZWYzXCIgOmljb249XCJNb3JlRmlsbGVkXCIgLz5cbiAgPC9lbC1zcGFjZT5cblxuICA8ZWwtdG91ciB2LW1vZGVsPVwib3BlblwiPlxuICAgIDxlbC10b3VyLXN0ZXAgOnRhcmdldD1cInJlZjE/LiRlbFwiIHRpdGxlPVwiVXBsb2FkIEZpbGVcIiA6bmV4dC1idXR0b24tcHJvcHM9XCJ7Y2hpbGRyZW46ICfkuIvkuIDmraUnfVwiPlxuICAgICAgPGltZ1xuICAgICAgICBzdHlsZT1cIndpZHRoOiAyNDBweFwiXG4gICAgICAgIHNyYz1cImh0dHBzOi8vZWxlbWVudC1wbHVzLm9yZy9pbWFnZXMvZWxlbWVudC1wbHVzLWxvZ28uc3ZnXCJcbiAgICAgICAgYWx0PVwidG91ci5wbmdcIlxuICAgICAgLz5cbiAgICAgIDxkaXY+UHV0IHlvdSBmaWxlcyBoZXJlLjwvZGl2PlxuICAgIDwvZWwtdG91ci1zdGVwPlxuICAgIDxlbC10b3VyLXN0ZXBcbiAgICAgIDp0YXJnZXQ9XCJyZWYyPy4kZWxcIlxuICAgICAgdGl0bGU9XCJTYXZlXCJcbiAgICAgIGRlc2NyaXB0aW9uPVwiU2F2ZSB5b3VyIGNoYW5nZXNcIlxuICAgIC8+XG4gICAgPGVsLXRvdXItc3RlcFxuICAgICAgOnRhcmdldD1cInJlZjM/LiRlbFwiXG4gICAgICB0aXRsZT1cIk90aGVyIEFjdGlvbnNcIlxuICAgICAgZGVzY3JpcHRpb249XCJDbGljayB0byBzZWUgb3RoZXJcIlxuICAgIC8+XG4gIDwvZWwtdG91cj5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgc2V0dXAgbGFuZz1cInRzXCI+XG5pbXBvcnQgeyByZWYgfSBmcm9tICd2dWUnXG5pbXBvcnQgeyBNb3JlRmlsbGVkIH0gZnJvbSAnQGVsZW1lbnQtcGx1cy9pY29ucy12dWUnXG5cbmltcG9ydCB0eXBlIHsgQnV0dG9uSW5zdGFuY2UgfSBmcm9tICdlbGVtZW50LXBsdXMnXG5cbmNvbnN0IHJlZjEgPSByZWY8QnV0dG9uSW5zdGFuY2U+KClcbmNvbnN0IHJlZjIgPSByZWY8QnV0dG9uSW5zdGFuY2U+KClcbmNvbnN0IHJlZjMgPSByZWY8QnV0dG9uSW5zdGFuY2U+KClcblxuY29uc3Qgb3BlbiA9IHJlZihmYWxzZSlcbjwvc2NyaXB0PlxuIiwiZWxlbWVudC1wbHVzLmpzIjoiaW1wb3J0IEVsZW1lbnRQbHVzIGZyb20gJ2VsZW1lbnQtcGx1cydcbmltcG9ydCB7IGdldEN1cnJlbnRJbnN0YW5jZSB9IGZyb20gJ3Z1ZSdcblxubGV0IGluc3RhbGxlZCA9IGZhbHNlXG5hd2FpdCBsb2FkU3R5bGUoKVxuXG5leHBvcnQgZnVuY3Rpb24gc2V0dXBFbGVtZW50UGx1cygpIHtcbiAgaWYgKGluc3RhbGxlZCkgcmV0dXJuXG4gIGNvbnN0IGluc3RhbmNlID0gZ2V0Q3VycmVudEluc3RhbmNlKClcbiAgaW5zdGFuY2UuYXBwQ29udGV4dC5hcHAudXNlKEVsZW1lbnRQbHVzKVxuICBpbnN0YWxsZWQgPSB0cnVlXG59XG5cbmV4cG9ydCBmdW5jdGlvbiBsb2FkU3R5bGUoKSB7XG4gIGNvbnN0IHN0eWxlcyA9IFsnaHR0cHM6Ly9jZG4uanNkZWxpdnIubmV0L25wbS9lbGVtZW50LXBsdXNAbGF0ZXN0L2Rpc3QvaW5kZXguY3NzJywgJ2h0dHBzOi8vY2RuLmpzZGVsaXZyLm5ldC9ucG0vZWxlbWVudC1wbHVzQGxhdGVzdC90aGVtZS1jaGFsay9kYXJrL2Nzcy12YXJzLmNzcyddLm1hcCgoc3R5bGUpID0+IHtcbiAgICByZXR1cm4gbmV3IFByb21pc2UoKHJlc29sdmUsIHJlamVjdCkgPT4ge1xuICAgICAgY29uc3QgbGluayA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2xpbmsnKVxuICAgICAgbGluay5yZWwgPSAnc3R5bGVzaGVldCdcbiAgICAgIGxpbmsuaHJlZiA9IHN0eWxlXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2xvYWQnLCByZXNvbHZlKVxuICAgICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdlcnJvcicsIHJlamVjdClcbiAgICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kKGxpbmspXG4gICAgfSlcbiAgfSlcbiAgcmV0dXJuIFByb21pc2UuYWxsU2V0dGxlZChzdHlsZXMpXG59XG4iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwidnVlXCI6IFwiaHR0cHM6Ly9jZG4uanNkZWxpdnIubmV0L25wbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vY2RuLmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL2Nkbi5qc2RlbGl2ci5uZXQvbnBtL2VsZW1lbnQtcGx1c0BsYXRlc3QvZGlzdC9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJodHRwczovL2Nkbi5qc2RlbGl2ci5uZXQvbnBtL2VsZW1lbnQtcGx1c0BsYXRlc3QvXCIsXG4gICAgXCJAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZVwiOiBcImh0dHBzOi8vY2RuLmpzZGVsaXZyLm5ldC9ucG0vQGVsZW1lbnQtcGx1cy9pY29ucy12dWVAMi9kaXN0L2luZGV4Lm1pbi5qc1wiXG4gIH0sXG4gIFwic2NvcGVzXCI6IHt9XG59IiwiX28iOnt9fQ==)

<img width="781" height="105" alt="image" src="https://github.com/user-attachments/assets/45a02fed-1f72-4add-b4c8-b01c51e476cd" />
